lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -119,6 +127,10 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :next_version,
                                        description: "Next version of the SDK",
                                        optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :automatic_release,
+                                       description: "If this is an automatic release",
+                                       optional: true,
                                        type: String)
```

Comment:
This is not actually a string right? It should be a boolean

Reply:
Good catch, updated